TemperatureControl/TemperatureControl.module.less

```diff
@@ -0,0 +1,21 @@
+// TemperatureControl.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.temperatureControl {
+	position: relative;
+
+	.slider {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		height: 100%;
+		width: 100%;
+	}
+
+	.applySkins({
+		height: @agate-temperaturecontrol-height;
+		width: @agate-temperaturecontrol-height;
```

Comment:
Let's define width and height based on the radius like ArcPicker

https://github.com/enactjs/agate/pull/340/files#diff-20ae5e7dc95d3df64abe101a0a76e461R181
```
		style: ({radius, style}) => {
			const size = ri.scaleToRem(radius * 2);
			return {...style, height: size, width: size};
		}
```

By doing it, we could remove all width and height styles in this file. I think it would be fine to add same margin or to remove any margin in TemperatureControl, WindDirectionControl, and FanSpeedControl.
CC: @stanca-pop-lgp , @adrian-cocoara-lgp

Reply:
ok, we will have the same margin for all components